lib/roo_on_rails/routemaster/publishers.rb

```diff
@@ -5,16 +5,16 @@ module Publishers
       @publishers = {}
 
       def self.register_default(publisher_class)
-        @default_publishers << publisher_class
+        @default_publishers << publisher_class.name
```

Comment:
Oops, actually here you want the class not the name of the publisher.

Unless you're going to store the publishers as names, and then do `constantize.new`